config/app/fixtures/shop_configuration.yaml

```diff
@@ -0,0 +1,236 @@
+                                translations:
+                                    en_US:
+                                        name: 'Category'
+                                    fr_FR:
+                                        name: 'Catégorie'
+
+                channel:
+                    options:
+                        custom:
+                            fashion_web_store:
+                                name: 'Fashion Web Store'
+                                code: 'FASHION_WEB'
+                                locales:
+                                    - '%locale%'
+                                currencies:
+                                    - 'EUR'
+                                enabled: true
+                                hostname: '%env(resolve:SYLIUS_FIXTURES_HOSTNAME)%'
+                                theme_name: '%env(resolve:SYLIUS_FIXTURES_THEME_NAME)%'
+                                shop_billing_data:
+                                    company: 'Sylius'
+                                    tax_id: '0001112222'
+                                    country_code: 'US'
+                                    street: 'Test St. 15'
+                                    city: 'eCommerce Town'
+                                    postcode: '00 33 22'
+                                menu_taxon: 'MENU_CATEGORY'
+                                contact_phone_number: '+41 123 456 789'
+                                contact_email: 'contact@example.com'
+
+                payment_method:
+                    options:
+                        custom:
+                            cash_on_delivery:
+                                code: 'cash_on_delivery'
+                                name: 'Cash on delivery'
+                                channels:
+                                    - 'FASHION_WEB'
+                            bank_transfer:
+                                code: 'bank_transfer'
+                                name: 'Bank transfer'
+                                channels:
+                                    - 'FASHION_WEB'
+                                enabled: true
+
+                shipping_method:
+                    options:
+                        custom:
+                            ups:
+                                code: 'ups'
+                                name: 'UPS'
+                                enabled: true
+                                channels:
+                                    - 'FASHION_WEB'
+                            dhl_express:
+                                code: 'dhl_express'
+                                name: 'DHL Express'
+                                channels:
+                                    - 'FASHION_WEB'
+                            fedex:
+                                code: 'fedex'
+                                name: 'FedEx'
+                                channels:
+                                    - 'FASHION_WEB'
+
```

Comment:
Set up these with specific zones; Otherwise, they'll get assigned at random, quite possibly invalidating going through checkout with some countries. The same issue popped up during QA of the plugin